openmc/settings.py

```diff
@@ -29,6 +29,178 @@ class RunMode(Enum):
 class Settings:
     """Settings used for an OpenMC simulation.
 
+    Parameters
+    ----------
```

Comment:
Instead of copy and paste, it might be better to simply put:
```Python
Parameters
---------------
**kwargs : dict
    Any keyword arguments are used to set attributes on the instance
```
That way, there's no risk of the two lists getting out of sync